cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/AnomalyDetails.java

```diff
@@ -15,6 +15,8 @@
 
 import static com.linkedin.cruisecontrol.CruiseControlUtils.utcDateFor;
 import static com.linkedin.kafka.cruisecontrol.detector.notifier.KafkaAnomalyType.GOAL_VIOLATION;
+import static com.linkedin.kafka.cruisecontrol.detector.notifier.KafkaAnomalyType.INTRA_BROKER_GOAL_VIOLATION;
+
```

Comment:
nit: redundant new lines. Just curious did you enable code style check? The code style should force to avoid 2 consecutive blank lines.

Reply:
I did, checkstyle passed (running the lastest version that exists in the default branch as well). Also, build process has a `checkStyle` task in it as well and I noticed this is a recurring thing in other java files as well (just following the status quo).
Let me know if it should be different.